pages/ksphere/kommander/1.2/release-notes/index.md

```diff
@@ -32,52 +31,55 @@ Kommander provides a command center for all your cloud native management needs i
 | **Maximum**        | 1.17.x  |
 | **Default**        | 1.17.3  |
 
-# Kommander 1.2.0 Beta 1 (September 14th, 2020)
+# New Features and Capabilities 
+
+
+## Support for air-gapped environment 
 
-## Features/Improvements
+## Improved catalog handling
 
-- Improved catalog handling
-- Improved Access Control for editing/deleting actions
+## Improved Access Control for editing/deleting actions 
 
-## Bug Fixes
+## Changed routing from hash-based to history-based to allow redirects after login
 
-- Fixed an issue where logout wasn't deleting any cookies
-- Smaller UX Bugs and Improvements
+GUI wizard to add customer repos 
+Conductor integration (e.g. LHS menu) 
+Catalog support for Helm repos 
+Kommander demo env 
+Support network tunnels
+app template  
+DKP product licensing 
+Show Correct Addon Status 
+Network Policy 
+Limit ranger 
+Kubefed scaling 
+Create EKS/etc clusters 
+Easy import process for 3rd party clusters
 
-## Component Versions
+* Added SAML IDP
+* Moved provisioning code to Konvoy, Kommander now uses Konvoy to provision clusters
+* Added a setting to configure thresholds for resource warnings 
+* Attaches EKS clusters now can show metrics
+* Added Infrastructure provider management on global level
```

Comment:
these should be move below something like "additional new feature" os something.

```suggestion
## Additional New Features
* Added SAML IDP
* Moved provisioning code to Konvoy, Kommander now uses Konvoy to provision clusters
* Added a setting to configure thresholds for resource warnings 
* Attaches EKS clusters now can show metrics
* Added Infrastructure provider management on global level
```

Reply:
Anything bulleted should go under fixed and improved issues. I moved these down.